### PR TITLE
hide "enabled" parameters on jenkins GUI

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.qaprosoft.jenkins.jobdsl.factory.pipeline
+
+@Grab('org.testng:testng:6.8.8')
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy]: remove @Grab

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/TestRailJobFactory.groovy [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.qaprosoft.jenkins.jobdsl.factory.pipeline
+
+@Grab('org.testng:testng:6.8.8')
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/TestRailJobFactory.groovy]: remove @Grab

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        this.name = name
+    }
+
+    def create() {
+        logger.info("QTestJobFactory->create")
+        def pipelineJob = super.create()
+        pipelineJob.with {
+            parameters {
+                configure stringParam('uuid', '', "uuid parameter")
+                configure addHiddenParameter('qtest_enabled', 'qtest_enabled boolean parameter', 'true')
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy]: why do we need it hidden? I recommend to remove it completely

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/TestRailJobFactory.groovy [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    def create() {
+        logger.info("TestRailJobFactory->create")
+        def pipelineJob = super.create()
+        pipelineJob.with {
+            parameters {
+                configure stringParam('uuid', '', "uuid parameter")
+                configure booleanParam('isRerun', false, 'isRerun boolean parameter')
+                configure addHiddenParameter('testrail_enabled', 'testrail_enabled boolean parameter', 'true')
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/TestRailJobFactory.groovy]: why do we need it hidden? I recommend to remove it completely

File: src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy (hunk 3 above)
Comment (ilya-varchenya) [reply src/com/qaprosoft/jenkins/jobdsl/factory/pipeline/QTestJobFactory.groovy]: We need to make this parameters true, because they have false by default

**Pull request merged**